doc: fix format issue in the doc (#566)
canonical/launchpad-manual: docs/user/how-to/projects/register-your-project.rst

-  offer an easy way for your project's users to post and answer support
   questions.

      .. tip::

>           If you want to register a translation or Ubuntu loco team in Launchpad, please create a new team, not a project.

Reviewer: This is over-indented. Check other admonitions/code blocks to see the correct level.

> Step 1: Log in to Launchpad

Reviewer: ```suggestion
Log in to Launchpad
```
No need to mention the step number. Apply to other headings below as well.

--------------------------- 
Log in to Launchpad and visit https://launchpad.net/projects/

Step 2: Register a project
-------------------------- 

> Click **Register a project** in the left-hand **Actions** menu.

Reviewer: ```suggestion
Select :guilabel:`Register a project` in the left-hand side menu.
```

> Step 3: Complete the project detail

Reviewer: ```suggestion
Step 3: Enter project details
```

----------------------------------- 

> Complete the ``Name``, ``Display Name`` and ``Title`` fields.

Reviewer: ```suggestion
Fill in the ``Name``, ``Display Name`` and ``Title`` fields.
```

Launchpad uses these three different names for your project in different
contexts:

-  **Name:** This is the short name that Launchpad will use for your
   project in URLs. For example: "blueprint" in

>    https://launchpad.net/blueprint for Launchpad's Blueprint

Reviewer: ```suggestion
   ``https://launchpad.net/blueprint`` for Launchpad's Blueprint
```
This should be in backticks because we don't want this sample link to be live.

   Tracker.
-  **Display name:** This is the name Launchpad uses for your project in
   the breadcrumb at the top of the screen. For example: "Blueprint".
-  **Title:** This is the long form of your project's name and is the
   most common name that Launchpad uses for your project. For example:
   "The Blueprint Feature Tracker".

Step 4: Review project summary and description
---------------------------------------------- 

> Your project summary and description are used on your project's homepage. A portion of the summary is also shown in search results.

Reviewer: Launchpad's documentation observes a line-length limit of 79 characters so this should wrap into the next line. You will find that different projects follow different conventions for this.

In this screen-shot of the Blueprint Tracker's project page, the summary

> is highlighted with a red line and the description with a blue line.

Reviewer: There is a missing screenshot here. You can leave this as is or you can replace it with a new one from a different project on Launchpad. `https://launchpad.net/blueprint` doesn't currently exist but there are plenty more including [Launchpad itself](https://launchpad.net/launchpad) and [Ubuntu](https://launchpad.net/ubuntu).

Author: Hi @odadacharles  Actually I made a screenshot on my own, but I am not sure if it's approriate.

<img width="1477" height="888" alt="Image" src="https://github.com/user-attachments/assets/a65ab067-d2fd-41ca-a575-b48875f4a974" />

I left it as original description first, and if you think the screenshot I made is suitable, I can help to upload it again.

Reviewer: This is perfect @williamlin-tw , you can add this. Make it shorter so we only see the top part, i.e., everything above `project information` since that's the relevant bit for this.

> Step 5: Identify the project group

Reviewer: ```suggestion
Identify the project group (optional)
```
Optional steps should be labelled as such so users quickly know to ignore them unless relevant to their situation.

---------------------------------- 
If your project is part of a wider project group, you can
tell Launchpad by entering the project group's name in the **Part of** text-box. If you're not sure of the project group's exact

> Launchpad name, click **Choose...** to search for it.

Reviewer: ```suggestion
If your new project is part of a wider project group, you can
tell Launchpad by changing the project's details. On the project overview page, select :guilabel:`Change details` and enter the project group's name in the :guilabel:`Part of` text-box. If you're not sure of the project group's exact Launchpad name, you can also search for it.
```
I updated this because this part is one step removed from the actual project creation process. Additionally, in at least one case, I decided not to be too specific with a UI element because these frequently change and it's not ideal to have to update the documentation whenever something small changes in the product.

For example: Blueprint is part of the launchpad-project group.

If you make this project part of a project group, the group preferences
and decisions around bug tracking, translation and security policy will
also apply to this project.

      .. note::

>           You can request a new project group by emailing feedback@launchpad.net. Describe your project, its constituent parts and why a project group is the best way to organise them. Launchpad team administrators will reply to let you know if your request has been successful.

Reviewer: Fix indentation

Step 6: Complete optional fields
-------------------------------- 
The following fields allow you to tell Launchpad users more
about your project and are not used by Launchpad itself.

      .. tip::

>           Once you've registered your project, you can use Launchpad to host your project's downloadable files. Find out more about :ref:`file downloads <making-your-project-files-available-for-download>`.

Reviewer: This step actually looks empty and is likely no longer relevant. Please remove it and move the `Tip` to the end of the doc after the last step.

Step 7: Add your project
------------------------ 

> When you're ready, click the **Add** button to create your project in Launchpad. Launchpad will now take you straight to your new project page.

Reviewer: ```suggestion
When you're ready, click the button to complete the registration. Launchpad will now take you straight to your new project page.
```
The UI now says a different thing. It says `Complete Registration` instead of `Add`. That's why it helps not to be too specific with UI elements.

   most common name that Launchpad uses for your project. For example:
   "The Blueprint Feature Tracker".

Review project summary and description
---------------------------------------------- 
Your project summary and description are used on your project's homepage. A portion of 
the summary is also shown in search results.

In this screen-shot of the Blueprint Tracker's project page, the summary
is highlighted with a red line and the description with a blue line.

Identify the project group (optional)
-------------------------------------- 
If your project is part of a wider project group, you can
tell Launchpad by changing the project's details. On the project overview page, select 
:guilabel:`Change details` and enter the project group's name in the :guilabel:`Part 
of` text-box. If you're not sure of the project group's exact Launchpad name, you can 
also search for it.

For example: Blueprint is part of the launchpad-project group.

If you make this project part of a project group, the group preferences
and decisions around bug tracking, translation and security policy will
also apply to this project.

>       .. note::

Reviewer: This is over indented

          You can request a new project group by emailing feedback@launchpad.net. 
          Describe your project, its constituent parts and why a project group is the best way to 
          organise them. Launchpad team administrators will reply to let you know if your request 
          has been successful.

Add your project
------------------------ 
When you're ready, click the button to complete the registration. Launchpad will now 
take you straight to your new project page.

>       .. tip::

Reviewer: This is also overindented

Reviewer: One second thought, once you've switched the two steps around, this tip can just be a paragraph with a title that says `Next steps`. You'll see something similar in some of the other docs.
